apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -163,11 +164,51 @@ class RosterDialog extends React.Component {
       classrooms.find(classroom => {
         return classroom.id === this.state.selectedId;
       }).name;
-
     this.props.handleImport(this.state.selectedId, selectedName);
     this.setState({selectedId: null});
   };
 
+  // create new function for redirect to NewEditPage
+  redirectToEditSectionPage = sectionId => {
+    const redirectUrl = '/sections/' + sectionId + '/edit';
+    window.location.href = redirectUrl;
+  };
+
+  // Creates the section and redirects to the edit page
+  handleRedirect = () => {
+    this.recordSectionSetupExitEvent(COMPLETED_EVENT);
+    const classrooms = this.props.classrooms;
+    const courseName =
+      classrooms &&
+      classrooms.find(classroom => {
+        return classroom.id === this.state.selectedId;
+      }).name;
+
+    const importSectionUrl =
+      this.props.rosterProvider === OAuthSectionTypes.google_classroom
+        ? '/dashboardapi/import_google_classroom'
+        : '/dashboardapi/import_clever_classroom';
+    const courseId = this.state.selectedId;
+
+    return new Promise((resolve, reject) => {
+      $.getJSON(importSectionUrl, {
```

Comment:
As a follow up, could we migrate this to use `fetch`? My understanding is that's the function we should be using for async calls now, plus I've never seen this pattern. My guess is it's old code.

Definitely can be a follow up though! No need to update this PR.